### DISCUSSION TITLE
clear Home and Grid labels from Utilities Dashboard

### DISCUSSION THREAD

**Discussion by bkbartk** [Blackymas/NSPanel_HA_Blueprint#1895] [Q&A] [answered]
How do I clear the Home and Grid labels form the utilities dashboard?
```
      utilities_page01_home_label: ' '
      utilities_page01_grid_label: ' '
```
Still shows Home and Grid.
I wanted to try a cleaner look with only the icons.

**Comment by edwardtfn** [comment, marked as answer]
I think I will have to change the TFT to allow this. I will take a look.

- #1899
